devops/big-dipper/docker-compose.yml

```diff
@@ -0,0 +1,24 @@
+version: '3'
```

Comment:
We just copy this to `/root` and it automatically gets picked up by the service script?
And the `.env` file as well?

I'd love to see that reference explicit.  I couldn't see this in any of the scripts.

Reply:
Maybe nicer to copy it to a "safer" location, as "/root" may have users mucking around. Somewhere, say, `/etc/bigdipper` would not